root-config/src/quipubyteltda-root-config.js

```diff
@@ -1,10 +1,14 @@
+import Vue from "vue";
+import VueRouter from "vue-router";
 import { registerApplication, start } from "single-spa";
 import {
   constructApplications,
   constructRoutes,
   constructLayoutEngine,
 } from "single-spa-layout";
 
+Vue.use(VueRouter);
```

Comment:
This is what fixes the problem

Reply:
@joeldenning Thank you! It does fix the problem.

I was trying to do that but I couldn't figure out how to do it.  Because, from my understanding, they (vue and vue-router) needed to be installed as dependecies. But when I did it, I got even more errors.

So, since vue and vue-router are registrered on importmap, is not needed to npm install them?